client/components/localized-moment/context.js

```diff
@@ -15,7 +15,7 @@ import getCurrentLocaleSlug from 'state/selectors/get-current-locale-slug';
 
 const debug = debugFactory( 'calypso:localized-moment' );
 
-const MomentContext = React.createContext( moment );
+const MomentContext = React.createContext( { moment, momentLocale: moment.locale() } );
```

Comment:
It might be worth mentioning in the body of this commit message _why_ this change was being made. It's notable because:

- It's a global calypso component and not directly related to any of your other changes.
- The default context value is only used when a provider is not present, so changing it is not often needed.

Reply:
I updated both the commit message and the PR description, I hope the change is clearer now. Thanks for the feedback!